website/docs/components/components-showcase/components-showcase.md

```diff
@@ -367,7 +367,7 @@ const App = function (props) {
   const items = Object.keys(group).map((el) => group[el]);
 
   return (
-    <Box style={cardsStyle}>
+    <Box tag="ul" style={cardsStyle}>
```

Comment:
discussed in DMs that would be nice to make so that the card (list item) is selected entirely, i.e. you can't select the link after selecting the entire card (now you can, and you basically have to go through each list item twice)